mon-pix/app/components/user-logged-menu.js

```diff
@@ -16,14 +16,17 @@ export default Component.extend(EmberKeyboardMixin, {
 
   keyboardActivated: true,
   _canDisplayMenu: false,
-  _user: null,
 
   canDisplayLinkToProfile: computed('routing.currentRouteName', function() {
     const currentRouteName = this.get('routing.currentRouteName');
 
     return currentRouteName !== 'profile' && currentRouteName !== 'board';
   }),
 
+  getIdentifier: computed('currentUser.user.email', function() {
```

Comment:
On ne met pas de verbe d'action (`get…`) dans le nom d'une propriété normalement, notamment parce que le calcul n'est pas nécessairement effectué au moment où on fait référence à la propriété.

```suggestion
  displayedIdentifier: computed('currentUser.user.email', function() {
```

Reply:
Done